src/EnergyPlus/Pumps.hh

```diff
@@ -122,131 +113,93 @@ namespace Pumps {
-              powerSizingMethod(PowerSizingMethod::SizePowerPerFlowPerPressure), powerPerFlowScalingFactor(348701.1), // 22 W/gpm
-              powerPerFlowPerPressureScalingFactor(1 / 0.78),                                                         // legacy impeller efficiency
-              MotorEffic(0.0), PumpEffic(0.0), FracMotorLossToFluid(0.0), Energy(0.0), Power(0.0), PartLoadCoef(4, 0.0), PressureCurve_Index(0),
-              PumpMassFlowRateMaxRPM(0.0), PumpMassFlowRateMinRPM(0.0), MinPhiValue(0.0), MaxPhiValue(0.0), ImpellerDiameter(0.0), RotSpeed_RPM(0.0),
-              RotSpeed(0.0), PumpInitFlag(true), PumpOneTimeFlag(true), CheckEquipName(true), HasVFD(false), OneTimePressureWarning(true),
-              HeatLossesToZone(false), ZoneNum(0), SkinLossRadFraction(0.0), LoopSolverOverwriteFlag(false)
-        {
-        }
     };
 
     struct ReportVars
     {
         // Members
-        int NumPumpsOperating;        // Used in pump bank. reports how many pumps are ON
-        Real64 PumpMassFlowRate;      // Mass flow rate of the pump
-        Real64 PumpHeattoFluid;       // Heat transfer from pump to fluid (W)
-        Real64 PumpHeattoFluidEnergy; // Pump Energy dissipated into fluid stream
-        Real64 OutletTemp;            // Pump outlet temperature
-        Real64 ShaftPower;            // Power input at the shaft
-        Real64 ZoneTotalGainRate;     // total pump skin losses to zone (W)
-        Real64 ZoneTotalGainEnergy;   // total pump skin losses to zone energy (J)
-        Real64 ZoneConvGainRate;      // pump skin losses convecting to zone air (W)
-        Real64 ZoneRadGainRate;       // pump skin losses radiating to inside of zone (W)
-
-        // Default Constructor
-        ReportVars()
-            : NumPumpsOperating(0), PumpMassFlowRate(0.0), PumpHeattoFluid(0.0), PumpHeattoFluidEnergy(0.0), OutletTemp(0.0), ShaftPower(0.0),
-              ZoneTotalGainRate(0.0), ZoneTotalGainEnergy(0.0), ZoneConvGainRate(0.0), ZoneRadGainRate(0.0)
-        {
-        }
+        int NumPumpsOperating = 0;          // Used in pump bank. reports how many pumps are ON
+        Real64 PumpMassFlowRate = 0.0;      // Mass flow rate of the pump
+        Real64 PumpHeattoFluid = 0.0;       // Heat transfer from pump to fluid (W)
+        Real64 PumpHeattoFluidEnergy = 0.0; // Pump Energy dissipated into fluid stream
+        Real64 OutletTemp = 0.0;            // Pump outlet temperature
+        Real64 ShaftPower = 0.0;            // Power input at the shaft
+        Real64 ZoneTotalGainRate = 0.0;     // total pump skin losses to zone (W)
+        Real64 ZoneTotalGainEnergy = 0.0;   // total pump skin losses to zone energy (J)
+        Real64 ZoneConvGainRate = 0.0;      // pump skin losses convecting to zone air (W)
+        Real64 ZoneRadGainRate = 0.0;       // pump skin losses radiating to inside of zone (W)
```

Comment:
One possible addition is to just eliminate this extra report vars struct entirely.  The variables can live on the base pump data structure, as they are both allocated to the same size every single time anyway.  No big deal, just noting for the future.